### PR TITLE
Update readme

### LINKED ISSUE #116: Update README.md to reflect conversion from compose to TS2.2 classes
**Enhancement** 

The README.md needs updating to reflect the conversion to TS2.2 class mixins.

### PR DESCRIPTION
**Pull request by maier49**
**Type:** docs

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**
Updates the README now that compose is no longer being used.
Resolves #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,65 +235,65 @@ observableStore.observe([ 'itemId', 'otherItemId' ]).subscribe(function() {
...
 	 * @param idTransform
 	 */
-	transform<V>(transformation: Patch<T, V> | ((item: T) => V), idTransform: string | ((item: V) => string)): MappedQueryTransformResult<V, S>;
+	transform<V>(transformation: Patch<T, V> | ((item: T) => V), idTransform: string | ((item: V) => string)): MappedQueryResultInterface<V, this>;
 }
 ```
 
 
-The result of querying or transforming will be a read only `QueryTransformResult`.
+The result of querying or transforming will be implement the `QueryResult` interface.
```
Comment (dylans) [inline README.md]: typo... "will be implement the..."

File: README.md [hunk 2]
```diff
@@ -305,19 +304,19 @@ export interface QueryTransformResult<T, S extends ObservableStore<any, any, any
...
+Unless the transform method is called without an idTransform, the result of any queries to a QueryableStore will be a `MappedQueryResult`, which includes additional data in its `StoreDelta` updates, and provides a `track()` method. The `StoreDelta` interface is extended by the `TrackedStoreDelta` interface which the `MappedQueryResult` provides to observers. This augments the interface by providing data indicating the current and previous indices of items that have been moved within, added to, or removed from, the view represented by the `MappedQueryResult`. Unlike `updates`, `deletes`, and `adds`, these properties are not related to specific operations, but instead just represent changes in the position of items within the collection.
```
Comment (dylans) [inline README.md]: Should idTransform and QueryableStore be in code quotes?

File: README.md [hunk 3]
```diff
@@ -340,20 +339,20 @@ export interface TrackableStoreDelta<T> extends StoreDelta<T> {
...
+As with the observable store mixin, the locally tracked data in a `MappedQueryResult` has the potential to become out of sync with the underlying storage. If the source `ObservableStore` has `fetchAroundUpdates` set to true, then any query transform results produced from it will only send up to date information to observers. If the source is not fetching around updates, the `track()` method provided as part of the `MappedQueryResult` interface can be used to create a copy of a `QueryResult` that will fetch after any updates from its source to make sure it has the latest data. `track()` produces a `TrackedQueryResult`, which has a `release()` method that provides a new, non-tracked query transform result.
```
Comment (dylans) [inline README.md]: Is Observable store still a mixin, I thought it was now an interface, so perhaps change As with the observable store mixin to As with the `ObservableStoreInterface`

**Review by dylans — CHANGES_REQUESTED** [review]
Overall looks great, a few minor nitpicks to address.

File: README.md (hunk 3 above)
Comment (maier49) [reply README.md]: Changed it to `ObservableStore`

**Review by dylans — APPROVED** [review]

**Pull request merged**